Sources/RugbyFoundation/Core/Common/Hashers/TargetsHasher.swift

```diff
@@ -84,10 +91,10 @@ final class TargetsHasher {
 extension TargetsHasher: ITargetsHasher {
     func hash(_ targets: TargetsMap, xcargs: [String], rehash: Bool) async throws {
         targets.modifyIf(rehash) { resetHash($0) }
-
+        let xcodeVersion = try xcodeCLTVersionProvider.version()
```

Comment:
Let's format Xcode version in one line?
```suggestion
        let xcodeVersion = try xcodeCLTVersionProvider.version()
        let formattedXcodeVersion = "\(xcodeVersion.base) (\(xcodeVersion.build ?? "unknown"))"
```

\
And then we can pass it like this:
```swift
// - xcode_version: 15.0.1 (15A507)
"xcode_version": formattedXcodeVersion
```

Reply:
Also, we can avoid using the `"unknown"` value:
```swift
let formattedXcodeBuildVersion = xcodeVersion.build.map { " (\($0))" } ?? ""
let formattedXcodeVersion = "\(xcodeVersion.base)" + formattedXcodeBuildVersion

// (15.0.1, 15A507) -> "15.0.1 (15A507)"
// (15.0.1, nil) -> "15.0.1"
```

Reply:
Applied!